.scripts/build_unix.sh

```diff
@@ -14,15 +26,30 @@ export "CONDA_BLD_PATH=$HOME/conda-bld/"
 curl -fsSL https://pixi.sh/install.sh | bash
 export PATH="$HOME/.pixi/bin:$PATH"
 
+if [[ "$target" == *"osx"* ]]; then
+    echo "osx"
+    export PATH=$(echo $PATH | tr ":" "\n" | grep -v 'homebrew' | xargs | tr ' ' ':')
+fi
+
+if [[ "$target" == "emscripten-wasm32" ]]; then
+    extra_channel="-c https://repo.mamba.pm/emscripten-forge -c nmarticorena"
```

Comment:
```suggestion
    extra_channel="-c https://repo.mamba.pm/emscripten-forge -c nmarticorena"
```

should we change this channel?

Reply:
Yes, apologies, my channel is no longer required.